test/package_analysis/basic.yaml

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
nit: there is no specification for it in dart... but in golang, people tend to through data used for testing into a testdata/ folder.

Consider a folder layout like:
```
lib/
  src/
    package_analysis/
      package_analysis.dart // exports everything you want to make publicly visible
      src/
        summary.dart
        ... // all the other files
test/
  package_analysis/
    package_analysis_test.dart
    testdata/
      basic.yaml
      ...
```